blackduck/HubRestApi.py

```diff
@@ -509,6 +509,12 @@ def get_vulnerable_bom_components(self, version_obj, limit=9999):
             logging.warning("Failed to retrieve vulnerable bom components for project {}, status code {}".format(
                 version_obj, response.status_code))
 
+    def get_Component_Remediation(self, bom_component):
```

Comment:
could you follow python style convention and use get_component_remediation instead of using capitals?

Reply:
Fixed